modules/kmm-symbolic-links-for-in-tree-dependencies.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-symbolic-links-for-in-tree-dependencies_{context}"]
+
+= Symbolic links for in-tree dependencies
+
+Some kernel modules depend on other kernel modules that are shipped with the node's operating system.
+To avoid copying those dependencies into the kmod image, KMM mounts `/usr/lib/modules` into both the build and the
```

Comment:
I think that we should define KMM in this first usage. 

```suggestion
To avoid copying those dependencies into the kmod image, Kernel Module Management  (KMM) mounts `/usr/lib/modules` into both the build and the
```

Reply:
added.